compiler-rt/test/builtins/Unit/trunctfxf2_test.c

```diff
@@ -28,7 +28,7 @@ char assumption_1[sizeof(long double) * CHAR_BIT == 128] = {0};
 #endif
 
 int main() {
-#if __LDBL_MANT_DIG__ == 64 && defined(__x86_64__) &&                          \
+#if __LDBL_MANT_DIG__ == 64 && (defined(__x86_64__) || defined(__i386__)) &&   \
     (defined(__FLOAT128__) || defined(__SIZEOF_FLOAT128__))
```

Comment:
```suggestion
#if defined(CRT_HAS_TF_MODE) && HAS_80_BIT_LONG_DOUBLE
```

Since we include int_lib.h should be possible to simplify this